sdk/src/main/java/com/horizen/sc2sc/CrossChainMessageHashSerializer.java

```diff
@@ -0,0 +1,20 @@
+package com.horizen.sc2sc;
+
+import scorex.util.serialization.Reader;
+import scorex.util.serialization.Writer;
+import sparkz.core.serialization.SparkzSerializer;
+
+public class CrossChainMessageHashSerializer implements SparkzSerializer<CrossChainMessageHash> {
+
+
```

Comment:
What about use the same approach of other Serializers that create a static serializer variable and return it with a `getSerializer` method